py3status/modules/keyboard_layout.py

```diff
@@ -63,11 +36,16 @@ def __init__(self):
         find the best implementation to get the keyboard's layout
         """
         try:
-            xkblayout()
+            self._xkblayout()
+            self.command = self._xkblayout
```

Comment:
setting `self.command` in the __init__ will make this a class method and will thus have py3status try to execute it directly like the `keyboard_layout` one. You can see this on the logs when running with -d flag :

`user method command failed (__xset() takes exactly 1 argument (3 given))`

To avoid this, just rename self.command to **self._command** please